distribution/lib/Standard/AWS/0.0.0-dev/src/Database/Redshift/Redshift_Details.enso

```diff
@@ -36,7 +37,7 @@ type Redshift_Details
        Attempt to resolve the constructor.
     resolve : Function -> Redshift_Details | Nothing
     resolve constructor =
-        Panic.catch Any (constructor:Redshift_Details) _->Nothing
+        Panic.catch Type_Error (constructor:Redshift_Details) _->Nothing
```

Comment:
Btw im wondering if we could extrac this into a common pattern:

```
resolve_constructor ctor x = Panic.catch Type_Error (ctor.from X) _->Nothing 
```
?

Reply:
its not a from conversion and I don't think we can dynamically add the `:<Type>` at runtime.

```
Execution finished with an error: Could not find a conversion from `Function` to `Filter_Condition`.
```

Reply:
As far as I know (@JaroslavTulach please tell me if I'm wrong), `(x:T)` is basically always equivalent to `T.from x`.

Reply:
>  `(x:T)` is basically always equivalent to `T.from x`.

I don't think so. Following snippet:
```ruby
x = 1
y = x : Number
```
has nothing to do with conversions. Only when the type-check fails, the _conversions path_ activates.